modules/opensearch/includes/opensearch.search.inc

```diff
@@ -601,3 +601,61 @@ function opensearch_search_object_ids($ids) {
   }
   return FALSE;
 }
+
+/**
+ * Auto-complete callback for the ting search block.
+ *
+ * The function returns JSON and ending execution.
+ *
+ * @param string $query
+ *   The string to search suggestion with.
+ *
+ * @return array
+ */
+function opensearch_search_autocomplete($query) {
+  $items = array();
+  $rest_url = variable_get('opensearch_search_autocomplete_suggestion_url', '');
+  $method = variable_get('ting_search_autocomplete_method', 'facets');
+  $settings = opensearch_search_autocomplete_settings();
+
+  if (empty($settings['index'])) {
+    watchdog('opensearch', 'Autocomplete index not defined ', array(), $severity = WATCHDOG_WARNING);
+    drupal_json_output(array());
+    return array();
+  }
+
+  // minimumString is not an openSuggestion parameter
+  unset($settings['minimumString']);
```

Comment:
`minimumString` should no longer be a part of `opensearch_search_autocomplete_settings()`. Consequently we do not need to unset it either.